app/uk/gov/hmrc/exports/services/mapping/declaration/AdditionalInformationBuilder.scala

```diff
@@ -34,8 +34,8 @@ class AdditionalInformationBuilder @Inject()() extends ModifyingBuilder[ExportsD
   override def buildThenAdd(model: ExportsDeclaration, declaration: Declaration): Unit = {
     val maybeLast3CharsOfGoodsLocation = model.locations.goodsLocation.flatMap(_.identificationOfLocation).map(_.takeRight(3).toUpperCase)
```

Comment:
Uhm... this does not seem right to me. What about
```
val maybeLast3CharsOfGoodsLocation = model.locations.goodsLocation.map(_.identificationOfLocation.takeRight(3).toUpperCase)
```

Reply:
The suggested doesn't work with the two optional properties